community/bolt/src/main/java/org/neo4j/bolt/v1/runtime/TransactionStateMachineSPI.java

```diff
@@ -64,7 +65,7 @@
     TransactionStateMachineSPI( GraphDatabaseAPI db,
                                 ThreadToStatementContextBridge txBridge,
                                 QueryExecutionEngine queryExecutionEngine,
-                                TransactionIdStore transactionIdStoreSupplier,
+                                Supplier<TransactionIdStore> transactionIdStoreSupplier,
```

Comment:
Have changed this to get `TransactionIdStore` from `db.getDependencyResolver()...`